scripts/build.sh

```diff
@@ -27,6 +28,7 @@ while [[ $# -gt 0 ]]; do
       ;;
     --ci)
       ci=true
+      #AHHHH we need to fix this at some point.
```

Comment:
What's AHHHH here?

Reply:
Ah, I don't remember if this was even me or not, but this was left as part of a reminder for the xports below it. Previously, we would use the default compiler, but since CI would default to GCC 9 at the time and `clang` defaulted to clang-9, we added the two export lines to ensure that we always use Clang 10 instead.

Reply:
This is removed.